.env

```diff
@@ -1,2 +1,5 @@
 REACT_APP_ORCID_REDIRECT=https://orcid.org/oauth/authorize?client_id=APP-IUN046ZZDWW0L6ZT&response_type=code&scope=/activities/update%20/read-limited&redirect_uri=https://useroffice.ess.eu/SignUp/
-GENERATE_SOURCEMAP=false
\ No newline at end of file
+GENERATE_SOURCEMAP=false
+
+# in dev
+REACT_APP_AUTH_ALLOWED_HOSTNAMES=localhost
```

Comment:
This will probably change after I figure out how to do this properly in different environments